storage/integration_test.go

```diff
@@ -2513,9 +2513,13 @@ func TestIntegration_Encryption(t *testing.T) {
 
 		checkRead("first object", obj, key, contents)
 
+		// We create 2 objects here and we can interleave operations to get around
+		// the rate limit for object mutation operations (create, update, and delete).
 		obj2 := client.Bucket(bucket).Object("customer-encryption-2")
+		obj4 := client.Bucket(bucket).Object("customer-encryption-4")
+
 		// Copying an object without the key should fail.
-		if _, err := obj2.CopierFrom(obj).Run(ctx); err == nil {
+		if _, err := obj4.CopierFrom(obj).Run(ctx); err == nil {
```

Comment:
looks like you can update lines 2526 - 2530 to use obj4 as well.

Reply:
That is true, but the objective here is to not use the same object consecutively if we can avoid it as the rate limit for objects is one create/update/delete per second. So IIUC L2562 will not hit that limit even if it occurs in the same second as 2522 - if we use the same object, it might.